netlify.toml

```diff
@@ -5,4 +5,4 @@
 [[redirects]]
   from = "/*"
   to = "/index.html"
-
+  status = 200
```

Comment:
This fixes the "deployed" site, enabling it to act as a SPA. It should have been done in #76.

I know it's an unrelated change, but figured we didn't need a PR just for this.